src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -537,6 +537,10 @@ A list of times, separated by semicolons, can be provided, e.g.
 brackets, and if more than one is given, the major time unit in each (hours
 or minutes) should be of the same type.
 
+.. note::
+
+   In this context ``T-00`` "-" is a placeholder for the hours column.
```

Comment:
Slightly confusing, the `T-` is a placeholder for hours in any context. Perhaps something like:

```suggestion
   ``T-00`` means every hour, on the hour. The ``-`` is a placeholder for the hours column.
```